openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -748,6 +749,8 @@ A Client makes a Credential Request to the Credential Endpoint by sending the fo
     * `alg`: REQUIRED. JWE [@!RFC7516] `alg` algorithm [@!RFC7518] for encrypting Credential Responses.
     * `enc`: REQUIRED. JWE [@!RFC7516] `enc` algorithm [@!RFC7518] for encrypting Credential Responses.
 
+`credential_configuration_id` or `format` parameters are used when the Credential issuance was requested using a `scope` value in the Authorization Request or a pre-authorisation code was used that did not return an Authorization Details.
```

Comment:
I very much appreciate the use of the British spelling of authorisation 😁 - but I suspect "authorized" was intended:

```suggestion
The `credential_configuration_id` or `format` parameters are used when the Credential issuance was requested using a `scope` value in the Authorization Request or a Pre-Authorized Code was used that did not return an Authorization Details.
```

Reply:
i also use british spelling for `diving licence` :D